apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/WriteTests.cs

```diff
@@ -297,6 +366,7 @@ public async Task WriteRandomBytes()
                 Assert.Equal(0, recordedValues.Count);
             }
         }
+        // [END spanner_test_write_read_random_bytes]
 
         [Fact]
```

Comment:
This one doesn't have START-END. It might have been on purpose though.

And there are some others in this file as well that doesn't have START-END (lines 241-261)

Reply:
Yup, this is only for tracking the tests that are required across multiple languages. We have some extra tests for ourselves.